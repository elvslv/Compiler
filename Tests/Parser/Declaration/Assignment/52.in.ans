type TT = INTEGER;
var A : array [const 1..const 10] of array [const 1..const 20] of REAL;
var B : array [const 1..const 10] of array [const 1..const 20] of REAL;
var I : TT;
var J : TT;

:=           
|            
|___[]       
|   |        
|   |___[]   
|   |   |    
|   |   |___A
|   |   |    
|   |   |___I
|   |        
|   |___J    
|            
|___[]       
    |        
    |___[]   
    |   |    
    |   |___B
    |   |    
    |   |___J
    |        
    |___I    
