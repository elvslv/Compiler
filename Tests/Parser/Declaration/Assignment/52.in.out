type TT = INTEGER;
var A : array [const 1..const 10] of array [const 1..const 20] of FLOAT;
var B : array [const 1..const 10] of array [const 1..const 20] of FLOAT;
var I : TT;
var J : TT;
[]   
|    
|___A
|    
|___I
